spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/src/main/java/org/springframework/cloud/stream/binder/kafka/config/KafkaBinderConfigurationProperties.java

```diff
@@ -17,110 +17,136 @@
 package org.springframework.cloud.stream.binder.kafka.config;
 
 import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.cloud.stream.binder.kafka.KafkaMessageChannelBinder;
+import org.springframework.cloud.stream.binder.kafka.KafkaMessageChannelBinder.Mode;
+import org.springframework.util.StringUtils;
 
 /**
  * @author David Turanski
+ * @author Ilayaperumal Gopinathan
  */
-@ConfigurationProperties(prefix = "spring.cloud.stream.binder.kafka.default")
- class KafkaBinderConfigurationProperties {
-	
-	private int batchSize;
-	
-	private long batchTimeout;
-	
-	private int requiredAcks;
-	
-	private int replicationFactor;
-	
-	private int concurrency;
-	
-	private String compressionCodec;
-	
-	private boolean autoCommitEnabled;
-	
-	private int fetchSize;
-	
-	private int minPartitionCount;
-	
-	private int queueSize;
+@ConfigurationProperties(prefix = "spring.cloud.stream.binder.kafka",
+		locations = "classpath:/META-INF/spring-cloud-stream/kafka-binder.properties")
```

Comment:
That was already removed via [this] (https://github.com/spring-cloud/spring-cloud-stream/commit/7ffe9070e3957f5097e8bb76289275a456c61dd8).